package.json

```diff
@@ -142,6 +142,7 @@
     "eslint-plugin-import": "^2.3.0",
     "eslint-plugin-node": "^5.0.0",
     "events": "^1.1.1",
+    "fibers": "^2.0.0",
```

Comment:
@berianjones this new dependency is still here. if we don't use it, it should be removed.